chart/templates/webserver/webserver-deployment.yaml

```diff
@@ -134,6 +142,17 @@ spec:
         - name: config
           configMap:
             name: {{ template "airflow_config" . }}
+        {{- if .Values.dags.persistence.enabled }}
+        - name: dags
+          persistentVolumeClaim:
+            claimName: {{ .Release.Name }}-dags
```

Comment:
If uses `dags.persistence.existingClaim`, it should be modified to 
`claimName: {{ .Values.dags.persistence.existingClaim }}`.
Currently, the webserver pod cannot find pvc, so it is put in a pending state.

Reply:
have raised a PR to fix this - https://github.com/apache/airflow/pull/9688